CHANGELOG.md

```diff
@@ -1,5 +1,79 @@
 # CHANGELOG
 
+## Release 2.4.4 (in development)
+
+### Features added
+- [#2561](https://github.com/scality/metalk8s/issues/2561) - install `kubectl`
+on all master nodes and cleanup old "bootstrap" pods after a bootstrap restore
+(PR [#2562](https://github.com/scality/metalk8s/pull/2562))
+
+### Security fixes
+- Due to critical vulnerabilities (
+[CVE-2020-11652](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-11652)
+and
+[CVE-2020-11651](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-11651))
+with CVSS score of 10.0 that was discovered affecting all `Salt Master`
+versions inferior to `3000.1`, this release ships with a `Salt Master` version
+updated to `3000.3`. Users, especially those who expose the `Salt Master` to
+the internet must therefore upgrade immediately.
+
+    [#650](https://github.com/scality/metalk8s/issues/650) - Upgrade Salt master
+    to version `3000.3`
+    (PR [#2549](https://github.com/scality/metalk8s/pull/2549))
+
+- [#2600](https://github.com/scality/metalk8s/issues/2600) - Upgrade Grafana to
+6.7.4 to handle CVE-2020-13379
+(PR [#2605](https://github.com/scality/metalk8s/issues/2605))
+
+### Enhancements
+- Add and allow `etcd` state to run health check by default
+(PR [#2147](https://github.com/scality/metalk8s/pull/2147))
+
+- Add pillar refresh to ensure deploy node is resilient
+(PR [#2147](https://github.com/scality/metalk8s/pull/2147))
+
+- [#2530](https://github.com/scality/metalk8s/issues/2530) - Fix cluster
+upgrade with deployed workloads failures due to disallowed disruptions
+(PR [#2531](https://github.com/scality/metalk8s/pull/2531))
+
+- Fix apiserver-proxy upstreams
+(PR [#2555](https://github.com/scality/metalk8s/pull/2555))
+
+- Add documentation on how to downgrade Salt if needed
+(PR [#2540](https://github.com/scality/metalk8s/pull/2540))
+
+- [#2589](https://github.com/scality/metalk8s/issues/2589) - Bump Kubernetes
+version to 1.15.12 (PR [#2595](https://github.com/scality/metalk8s/pull/2595))
+
+### Bug fixes
+
+- [#2029](https://github.com/scality/metalk8s/issues/2029) - Provide support
+for python-kubernetes client `v11`
+(PR [#2554](https://github.com/scality/metalk8s/pull/2554))
+
+- [#2444](https://github.com/scality/metalk8s/issues/2444) - Fix SLS rendering
+missing pillar key and add pillar check in etcd orchestrate
+(PR [#2445](https://github.com/scality/metalk8s/pull/2445))
+
+- [#2524](https://github.com/scality/metalk8s/issues/2524) - Fix salt-minion
+upgrade and downgrade during orchestrate
+(PR [#2525](https://github.com/scality/metalk8s/pull/2525))
+
+- [#1703](https://github.com/scality/metalk8s/issues/1703) - Read version
+details from buildchain
+(PR [#2433](https://github.com/scality/metalk8s/pull/2433))
+
+- [#2551](https://github.com/scality/metalk8s/issues/2551) - Fix invalid check
+in downgrade precheck
+(PR [#2563](https://github.com/scality/metalk8s/pull/2563))
```

Comment:
Remove this one it's a bug introduced during the development and fixed before releasing any version